torchci/components/JobSummary.tsx

```diff
@@ -1,11 +1,16 @@
 import { JobData } from "lib/types";
 import JobConclusion from "./JobConclusion";
 
-export default function JobSummary({ job }: { job: JobData }) {
+export default function JobSummary({ job, highlight }: { job: JobData; highlight: boolean; }) {
   return (
     <>
       <JobConclusion conclusion={job.conclusion} />
       <a href={job.htmlUrl}> {job.jobName} </a>
+      {highlight ? <b> [{job.branch}] </b> : <span>[{job.branch}]</span> }
```

Comment:
One more bug: This component is shared with other pages, and those pages aren't setup to receive the branch (and they don't need to be).

However, the branch being null there is resulting in those pages having empty brackets:

Example: https://torchci-git-gh-daizeng19841head-fbopensource.vercel.app/pytorch/pytorch/commit/1da5aeb97b73664ff0fe2f4bb48379655cede969

<img width="873" alt="image" src="https://user-images.githubusercontent.com/4468967/198718709-bedcc1d4-0d7c-4c19-8234-0f5b080e3358.png">

Reply:
Thanks for catching that! I've added the null/empty check for job.branch